src/blueapi/cli/cli.py

```diff
@@ -37,11 +36,21 @@
 @click.option(
     "-c", "--config", type=Path, help="Path to configuration YAML file", multiple=True
 )
+@click.option("--env-prefix", default="APP_", help="Prefix for environment variables")
```

Comment:
Should: I think the prefix should default to `BLUEAPI`, `APP` is too generic.

Reply:
I don't think it'd be configurable, not sure

Reply:
deleted